.lintstagedrc

```diff
@@ -1,8 +1,8 @@
 {
-  "app/**/*.{js,jsx,json,css,scss,graphql,gql}": [
+  "app/**/*.{js,jsx,json,graphql,gql}": [
     "prettier --write",
   ],
-  "api/**/*.{js,jsx,json,css,scss,graphql,gql}": [
+  "api/**/*.{js,jsx,json,graphql,gql}": [
```

Comment:
You only have to ignore the theme css files not everything.

Reply:
Okay